image.yaml

```diff
@@ -27,7 +27,8 @@ envs:
     - name: HTTPS_ENABLE_HTTP2
       value: "true"
     - name: "JAVA_OPTS_APPEND"
-      example: "-Dfoo=bar"
+      value: "-Dee8.preview.mode=false"
```

Comment:
It shouldn't be necessary to set this to false, as that's the default behavior if the prop is not set.

Reply:
Also, if we forget to remove this when we're EE 8 certified and -Dee8.preview.mode no longer has any effect at all, we will be misleading users into perhaps thinking the no-longer present EE 7 API jars can still be used.